dashboard/test/ui/features/teacher_tools/projects/public_project_gallery_project_validator.feature

```diff
@@ -29,40 +29,6 @@ Scenario: Published Projects Show In Recency Order
   Then I debug element ".ui-project-name-playlab:eq(3)" text content
   Then element ".ui-project-name-playlab:contains('Published'):eq(0)" contains text "Newer Published"
 
-Scenario: Featured Projects Show Before Published Projects
```

Comment:
Removing these two scenarios since we will be removing 'published' projects soon.
Also, project validators now 'save' a project as a featured project from the admin UI and can feature it as a currently featured project at 'projects/featured'.